### ISSUE TITLE
New Icons

### ISSUE THREAD

**Issue by dcruz112** [rudrab/Shadow#126]
I've been using shadow for a while and really love it! I have a few icons to request. I've attached them [here](https://github.com/rudrab/Shadow/files/1744587/iconrequest.tar.gz) (I'm mostly interested in TeamViewer).

There are also some Ubuntu icons missing (namely the language and Wi-Fi icons, which I've taken a screenshot of here).

![shadow](https://user-images.githubusercontent.com/4485335/36487416-cfdacc8a-16ee-11e8-8675-74ccad76aa82.jpg)

**Comment by rudrab** [comment]
Thanks a lot. I will update those icons very soon.

**Comment by rudrab** [comment]
Hi,
sorry for the delay...actually i didnt had a ubuntu installation to check...I did a virtual machine to find the icons but I am nit sure if this solves the problem, as there are many icons with given name. So, please pull it from github and check if this solves the internet access (nm-signal) problem. 
I am on my way to generate the language icons.

Kindly let me know.

**Comment by dcruz112** [comment]
Hi,

Thanks for doing this project! The Wi-Fi icon is now fixed as seen below:
[image: signal.jpg]

The 'connecting' animation is the same though
[image: searching.jpg]

Best,
David

On Thu, Mar 15, 2018 at 7:08 AM rudrab <notifications@github.com> wrote:e

**Comment by dcruz112** [comment]
Also it looks like the ethernet or disconnected icons are still the same

[image: disconnected.jpg]

**Comment by rudrab** [comment]
hi, the images are not available....can you please repost?

**Comment by dcruz112** [comment]
I hate to be a bugbear, but I pulled and the animation hasn't updated (even
after restarting). If you've been working in a VM, then it might be better
to open a separate issue: there are system icons in Ubuntu that don't use
your icon set.

Also, the icons I included in the zip folder haven't been updated (though I
now realize some of those, e.g., nm-device-wireless.png, are included in
the system icons that remain unchanged).
